gulp/tasks/rollup.js

```diff
@@ -8,6 +9,7 @@ const uglify = require('../common/gulp-uglify.js');
 
 const DEV = !!JSON.parse(process.env.BUILD_DEV || 'true');
 const DEMO = !!JSON.parse(process.env.BUILD_DEMO || 'false');
+const VERSION = fs.readFileSync('setup.py', 'utf8').match(/\d{4}[^']*/)[0] || '0';
```

Comment:
let's raise an error if the match cannot be found. That way it won't silently break.

Reply:
Done